config/routes.rb

```diff
@@ -0,0 +1,24 @@
+Rails.application.routes.draw do
+  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  get 'users/login_form'
+  get 'users/login'
```

Comment:
Are these meant to be here? I don't believe these are doing anything without a `to` argument.